.ci/common.sh

```diff
@@ -20,6 +20,8 @@ function bumpPodsInfo() {
     TARGET_DIR="${ARTIFACT_DIR}/${NS}-info"
     mkdir -p "$TARGET_DIR"
 
+    oc get pods -n ${NS}
```

Comment:
Is it really useful? Maybe then at least separate it from everything else with something like
```
echo "===== Bumpings pods logs in namespace $NS. Pods: ======"
oc get pods -n ${NS}
```

Reply:
It helps to quickly find pods status when build fail.
For example, part of `build-log.txt` from last build:
```
+ TARGET_DIR=/logs/artifacts/devworkspace-che-info
+ mkdir -p /logs/artifacts/devworkspace-che-info
+ oc get pods -n devworkspace-che
NAME                                       READY   STATUS             RESTARTS   AGE
devworkspace-che-manager-768b7dd9c-gkj57   0/1     CrashLoopBackOff   8          19m
```